src/arch/src/aarch64/fdt.rs

```diff
@@ -82,12 +82,12 @@ pub enum Error {
 type Result<T> = result::Result<T, Error>;
 
 /// Creates the flattened device tree for this aarch64 microVM.
-pub fn create_fdt<T: DeviceInfoForFDT + Clone + Debug>(
+pub fn create_fdt<T: DeviceInfoForFDT + Clone + Debug, S: ::std::hash::BuildHasher>(
```

Comment:
I'm just curios why we need to add `S: ::std::hash::BuildHasher`. What's the clippy warning here ?

Reply:
This is the one https://rust-lang.github.io/rust-clippy/master/index.html#implicit_hasher

Reply:
Ok. `::std::...` looks strange. Can we use simply `std::` or `super::std::...` ?

Reply:
Done, take a look!